packages/react-day-picker/src/components/Table/utils/getWeeks.test.ts

```diff
@@ -0,0 +1,33 @@
+import { enGB, enUS } from 'date-fns/locale';
+import { getWeeks } from './getWeeks';
+
+describe('It should render weeks at the boundaries of years correctly', () => {
+  test('If the first days of the month start at week 52 from the previous year, week 52 should be rendered on the 1st row', () => {
```

Comment:
We should update the tests to follow this format:

```ts
describe('when the first days of the month start at week 52 from the previous year', () => {
  /* some setup here */
  test('the 1st row should render 52', () => {
    expect(); // etc
```

Reply:
I'll try to have just one expect per test

Reply:
Yeah! We should improve the test to be more descriptive, we can do that during some clean-up…